pom.xml

```diff
@@ -46,7 +46,7 @@
         <revision>3.1.3</revision>
         <changelist>-SNAPSHOT</changelist>
         <hpi.compatibleSinceVersion>3.0.0</hpi.compatibleSinceVersion>
-        <jenkins.version>2.319.3</jenkins.version>
+        <jenkins.version>2.346.3</jenkins.version>
```

Comment:
Thia version requires java 11 or newer

Reply:
2.361.1 was the first LTS to [require Java 11](https://www.jenkins.io/blog/2022/06/28/require-java-11/)

Reply:
Notable changes since 2.346.3:

    Require Java 11 or newer. ([Blog post](https://www.jenkins.io/blog/2022/06/28/require-java-11/), [issue 68570](https://issues.jenkins.io/browse/JENKINS-68570), [JEP-236](https://github.com/jenkinsci/jep/blob/master/jep/236/README.adoc), [pull 6083](https://github.com/jenkinsci/jenkins/pull/6083))

https://www.jenkins.io/changelog-stable/